libvast/src/system/node.cpp

```diff
@@ -322,7 +325,8 @@ node_state::spawn_command(const command::invocation& invocation,
     label = *label_ptr;
   } else {
     label = comp_name;
-    const char* multi_instance[] = {"importer", "exporter", "source", "sink"};
+    const char* multi_instance[]
+      = {"importer", "explorer", "exporter", "source", "sink"};
```

Comment:
Why does the explorer have to be in here when the pivoter is not?

Reply:
Good question - did we ever try to spawn to pivoters in parallel? Maybe we forgot to add it but  never noticed.